posts/dialog.md

```diff
@@ -0,0 +1,13 @@
+feature: <dialog>
+status: caution
+tags: polyfill
+kind: html
+polyfillurls:[dialog-polyfill](https://github.com/GoogleChrome/dialog-polyfill)
+
+The `<dialog>` element can be used to create popup dialogs or modal dialogs for web applications, and is part of [HTML 5.1)](http://www.w3.org/html/wg/drafts/html/master/interactive-elements.html#the-dialog-element).
```

Comment:
Seems like the link isn't totally right yet. There's still an extra bracket in inside the square ones.